src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx

```diff
@@ -1,13 +1,109 @@
-import { Flex, Text, WinningBidIcon } from "@artsy/palette"
+import {
+  Button,
+  Clickable,
+  Flex,
+  ModalDialog,
+  Text,
+  WinningBidIcon,
+} from "@artsy/palette"
+import { toTitleCase } from "@artsy/to-title-case"
+
+import { useState } from "react"
 import { RouterLink } from "System/Router/RouterLink"
+import { useFeatureFlag } from "System/useFeatureFlag"
+import { Media } from "Utils/Responsive"
+
+// TODO:- We are using displayText for Statuses for now. Consider changing the logic when proper statuses are made available on Metaphysics.
+// See https://artsyproduct.atlassian.net/browse/SWA-217
+// same as on Eigen
+const STATUSES: { [key: string]: { color: string; text: string } } = {
+  "submission in progress": { color: "yellow150", text: "In Progress" },
+  "submission evaluated": { color: "orange150", text: "Evaluation Complete" },
+  sold: { color: "black100", text: "Artwork Sold" },
+}
```

Comment:
**suggestion:** These statuses should better be coming from Metaphysics IMO (we can store the color as a hex code as well). I know this logic exists as well in Eigen so I would recommend addressing this in a different ticket.

Reply:
Yes, it thought of addressing it in a different ticket